src/ICeWall/src/ICeWall.tcl

```diff
@@ -2682,6 +3010,8 @@ namespace eval ICeWall {
 
     set corner_size [get_footprint_corner_size]
     set bump_master [get_cell bump]
+    lassign [$bump_master getOrigin] master_x master_y
+
     for {set row 1} {$row <= $num_bumps_y} {incr row} {
       for {set col 1} {$col <= $num_bumps_x} {incr col} {
         set origin [get_bump_origin $row $col]
```

Comment:
on line 3029, you use the origin to place the "pin", but this does not account for the offset form the master.  I think this is why our pins are appearing at the bottom left corner of the bump instance, instead of the center of the pin.

Reply:
Use get_bump_centre to place the pin, instead of get_bump_origin